ipc/src/server.rs

```diff
@@ -219,10 +235,32 @@ impl<M: Metadata, S: Middleware<M>> ServerBuilder<M, S> {
 			Err(e) => Err(e)
 		}
 	}
+
+	/// Creates the underlying named pipes with `READ_GEENRIC | FILE_WRITE_DATA` access permissions to Everyone.
+	#[cfg(windows)]
+	pub fn allow_everyone_to_connect(&mut self) -> ::std::io::Result<()> {
+		self.security_attributes = Some(SecurityAttributes::allow_everyone_connect()?);
+		Ok(())
+	}
+
+	/// Creates the underlying named pipes with `READ_GEENRIC | WRITE_GENERIC` access permissions to Everyone.
+	#[cfg(windows)]
+	pub fn allow_everyone_to_create_pipes(&mut self) -> ::std::io::Result<()> {
+		self.security_attributes = Some(SecurityAttributes::allow_everyone_create()?);
+		Ok(())
+	}
+
+	/// Creates the underlying named pipes with `READ_GEENRIC | WRITE_GENERIC` without specifying any security
+	/// attributes.
+	#[cfg(windows)]
+	pub fn default_security(&mut self) {
```

Comment:
ditto, let's leave it in parity-tokio-ipc

Reply:
So, you think that `parity_tokio_ipc::SecurityAttributes` should still be exposed to the consumers of of `jsonrpc_ipc_server::ServerBuilder`? Just trying to understand the end goal here :)